core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.0;
+
+pragma experimental ABIEncoderV2;
+
+import "../../tokenized-derivative/AddressWhitelist.sol";
+import "../../tokenized-derivative/ContractCreator.sol";
+import "../../common/Testable.sol";
+import "./ExpiringMultiParty.sol";
+
+/**
+@title Expiring Multi Party Contract creator
+@notice Factory contract to create and register new instances of expiring multiparty contracts
+*/
+contract ExpiringMultiPartyCreator is ContractCreator, Testable {
+    constructor(bool _isTest, address _finderAddress) public ContractCreator(_finderAddress) Testable(_isTest) {}
```

Comment:
Do you think TokenFactory address should be saved at the creator level? No strong opinion either way, just wanted to raise the point.

Reply:
It could be but right now all children (`ExpiringMultiParty`) store their own instance of the `TokenFactory` within them. This could be pushed up a level to the creator but then how would the `ExpiringMultiParty` know about the `TokenFactory` if it is stored within it's parent? 

This would also mean that the `ExpiringMultiPartyCreator` would be responsible for creating the new token from the `TokenFactory` and then passing it to the `ExpiringMultiParty` as a parameter 🤔. This pattern might make sense. Other thoughts @mrice32 and @nicholaspai ?

Reply:
This we could do but then it will place all logic in creating a new token within the `ExpiringMultiPartyCreator` and then we would need to feed the address of the new token down to the `ExpiringMultiParty`. Dont have any issues with doing this but right now it feels a bit cleaner with this occurring at the `ExpiringMultiParty` level. With that said, it would simplify the creator of `ExpiringMultiParty` a bit more to bump this upstream. Other thoughts @nicholaspai and @mrice32?

Reply:
Not super opinionated here, but IIUC, I think the suggestion is to still do everything the same in the EMP contract, but just not require the user to provide that address when calling `createExpiringMultiParty`. IMO, it might be nice to provide as few parameters as possible to simplify the UX around creating a new contract.